submissions/BogdanGapon/HTML & CSS popup/index.html

```diff
@@ -0,0 +1,132 @@
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/blogs.png" alt="Blog"/>Blog
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/calendar.png" alt=""/>Calendar
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/chat.png" alt=""/>Chat
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/classroom.png" alt=""/>Classroom
+                      </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/contacts.png" alt="" />Contacts
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/documents.png" alt=""/>Documents
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/files.png" alt=""/>Files
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/mail.png" alt=""/>Mail
+                    </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/maps.png" alt=""/>Maps
+                      </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/mars.png" alt=""/>Mars
+                      </a>
+                  </li>
+                  <li class="popup-list__list">
+                    <a href="">
+                      <img src="./img/photos.png" alt=""/>Contacts
+                    </a>
+                  </li>
+                </ul>
+
+                <input
+                  class="checkbox-more visually-hidden"
+                  type="checkbox"
+                  name="checkbox_2"
+                  id="labe_id"
+                />
+                <label for="labe_id" class="label-more is-hidden">More</label>
+                <div class="next-popup-wrapper">
+                  <ul class="popup-item">
+                    <li class="popup-list__list">
+                      <a href="">
+                        <img src="./img/promo.png" alt=""/>Video
+                        </a>
+                    </li>
+                    <li class="popup-list__list">
+                      <a href="">
+                        <img src="./img/search.png" alt=""/>Search
+                        </a>
```

Comment:
Sorry, bro. Lets  close the issue about width and height for img tag. Have serched about this  and this is my bad. Please revert  all  changes related to this comment. You really made searching good job.
https://jakearchibald.com/2022/img-aspect-ratio/

Reply:
it`s okay, thank you ) i revert all changes related to this comment.